### PR TITLE
[NCP-3449] added text related to the retry verification

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Comment by Jianghai-Quantil** [comment]
@jungho73 As mentioned earlier in a separate pull request, please wait for review by me or Bin for changes to Chinese texts. Please do not haste to merge. 

The changes to Chinese texts in this pull request are acceptable, but can be improved. Since it is already merged, I'll leave it as it is.

**Pull request merged**
